docker/dockerfiles/Dockerfile_be_centos

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN yum install -y tzdata openssl curl vim ca-certificates fontconfig gzip tar mysql java-11-openjdk
+
+COPY /be /opt/starrocks/be/
+Copy be_entrypoint.sh be_prestop.sh /opt/starrocks/
```

Comment:
these scripts are not available in starrocks repo. It is a little weird to check in a partial done works into starrocks repo.